### PR TITLE
add the registry-qps kubelet flag

### PR DESCRIPTION
**Pull request by sp-joseluis-ledesma**

### CODE REVIEW

File: pkg/apis/kops/v1alpha2/componentconfig.go [hunk 1]
```diff
@@ -179,6 +179,8 @@ type KubeletConfigSpec struct {
 	AuthenticationTokenWebhook *bool `json:"authenticationTokenWebhook,omitempty" flag:"authentication-token-webhook"`
 	// AuthenticationTokenWebhook sets the duration to cache responses from the webhook token authenticator. Default is 2m. (default 2m0s)
 	AuthenticationTokenWebhookCacheTTL *metav1.Duration `json:"authenticationTokenWebhookCacheTtl,omitempty" flag:"authentication-token-webhook-cache-ttl"`
+	// RegistryQPS if > 0, limit registry pull QPS to this value.  If 0, unlimited. (default 5)
+	RegistryQPS *int32 `json:"registryQPS,omitempty" flag:"registry-qps"`
```
Comment (justinsb) [inline pkg/apis/kops/v1alpha2/componentconfig.go]: Could we make this ` RegistryPullQPS *int32 `json:"registryPullQPS,omitempty" flag:"registry-qps"``

That is what the kubelet component config is going to be https://github.com/kubernetes/kubernetes/blob/d5a3db003916b1d33b503ccd2e4897e094d8af77/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L210

**Comment by maruina** [comment]
Hey @sp-joseluis-ledesma, thanks for the PR. We're interested in this as well, would you mind to also add the `RegistryBurst` flag as well? I appreciate if you don't have time to follow up on this and happy to pick the PR up.

**Comment by sp-joseluis-ledesma** [comment]
I have renamed RegistryQPS to RegistryPullQPS + added RegistryBurst

**Comment by maruina** [comment]
Thanks a lot!

**Comment by sheldonkwok** [comment]
I don't mind forking and rebasing to get the ball rolling again. The qps is way too low by default and we need to up it.

**Comment by sp-joseluis-ledesma** [comment]
Me neither :) but I did that once, and if it is not going to be merged soon we will have to do it again @justinsb , @mikesplain If I do rebase this it will be merged soon ? ^^

thanks!

**Comment by mikesplain** [comment]
Looks good and looks like Justin's comments were addressed. Thanks @sp-joseluis-ledesma!

/lgtm
/approve

**Comment by ommehaz** [comment]
Is registryPullQPS related to this error “pull QPS exceeded”. If it is how to disable it, can a kindly soul explain ? Thanks 🙏🏻

**Pull request merged**